docs/panels.md

```diff
@@ -21,5 +21,21 @@ Panels can be added using the menu or keyboard shortcuts.
 
 Panels can be reordered simply by dragging them into the desired position.
 
-- Click on the space next to the `x` button and move the panel to the left or right.
+- Click anywhere on the head or borders of the panel and drag it to the left or right.
 
+## Renaming Panels <a name="rename"></a>
+
+You can edit panel names in two ways:
+
+- Double click the current panel name.
+- Click the `pencil` icon next to the `x` button at the top of the panel.
```

Comment:
'Type the new name' step is missing.
Reorganize into a better flow?
Instead of stating various ways various things can be done, explain how to change the name (the basic flow) step by step, while mentioning extra info (e.g. how to abort) as along the way.